windows/src/robocorp/windows/__init__.py

```diff
@@ -114,13 +114,50 @@ def config() -> "Config":
     return __get_cache_mod().config()
 
 
+@overload
+def find_window(
+    locator: Locator,
+    search_depth: int = ...,
+    timeout: Optional[float] = ...,
+    wait_time: Optional[float] = ...,
+    foreground: bool = ...,
+    raise_error: Literal[True] = ...,
+) -> "WindowElement":
+    ...
+
+
+@overload
+def find_window(
+    locator: Locator,
+    search_depth: int = ...,
+    timeout: Optional[float] = ...,
+    wait_time: Optional[float] = ...,
+    foreground: bool = ...,
+    raise_error: Literal[False] = ...,
+) -> Optional["WindowElement"]:
+    ...
+
+
+@overload
+def find_window(
```

Comment:
When is not passed explicitly, it is inferred from the fact that it is assumed to be "passed" with `True` implicitly. At least **mypy** doesn't scream in this case. Let me know if you think otherwise and if there's any trouble with the actual change, as I'm trying to not add unnecessary code.